cli/command/command.go

```diff
@@ -69,8 +76,20 @@ func PortCommand(factory app.ProjectFactory) cli.Command {
 // UpCommand defines the libcompose up subcommand.
 func UpCommand(factory app.ProjectFactory) cli.Command {
 	return cli.Command{
-		Name:   "up",
-		Usage:  "Bring all services up",
+		Name:      "up",
+		Usage:     "Builds, (re)creates, starts, and attaches to containers for a service.",
+		ArgsUsage: "[SERVICE...]",
+		Description: "Unless they are already running, this command also starts any linked services.\n\n" +
+			"The `libcompose-cli up` command aggregates the output of each container. When\n" +
```

Comment:
Thinking if GoDoc can be used somehow, I know @calavera did some experimenting in this area in this PR https://github.com/docker/docker/pull/16125. However (obviously) that's used to generate external documentation, not for usage in the output of the software itself